projects/hslayers/components/layer-manager/dialogs/copy-layer-dialog.component.html

```diff
@@ -3,13 +3,13 @@
         <div class="modal-content">
             <div class="modal-header">
                 <h4 class="modal-title">
-                    {{data.title}}
+                    {{data.title | translateHs}} ?
```

Comment:
Why the question mark?
EDIT: Oh, I can see that is because the title is actually a question... In that case, I am in favour of having that as a different string for translation. Albeit this approach may be OK for languages we support now, it would quickly fail for others (like Spanish, not even taking the RTL or non-latin languages into consideration).

Reply:
I mean...i would not bother going into such details

Reply:
Ack. You are right. "Nehas, co tě nepálí" approach is fine.